lib/csu/common/notes.h

```diff
@@ -27,7 +27,8 @@
 #ifndef	CSU_COMMON_NOTES_H
 #define	CSU_COMMON_NOTES_H
 
-#define NOTE_FREEBSD_VENDOR	"FreeBSD"
+#define NOTE_FREEBSD_VENDOR		"FreeBSD"
```

Comment:
Not sure why this changed?

Reply:
I wanted the values to vertically align. If we prefer to reduce differences, I can revert that.

Reply:
You're right. They did but my vim setup somehow broke. After executing `:exec FreeBSD_Style()` they aligned correctly. Thanks.